package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "ballerina-extension",
+    "title": "Ballerina Brackets Extension",
+    "description": "File Format Support for WSO2 Ballerina Development Environment (http://ballerinalang.org/)",
```

Comment:
How about "Syntax Highlighting Support for WSO2 Ballerina (http://ballerinalang.org/)" ?